src/components/DefineDissolution/DestroyCertificate.vue

```diff
@@ -0,0 +1,91 @@
+<template>
+  <div>
```

Comment:
Root Id is sometimes helpful as well as for consistency with our other components. ie `destroy-certificate`. 
But not required!

Reply:
Yes, please add an id (or classname if this component can have multiple instances on the same page).